armi/utils/dochelpers.py

```diff
@@ -68,6 +68,49 @@ def create_table(rst_table, caption=None, align=None, widths=None, width=None):
     return "\n".join(rst)
 
 
+def createListTable(rows, caption=None, align=None, widths=None, width=None):
```

Comment:
can you add a `classes=None` arg to fill in the `:class:` attr of the list table? This would be nice to e.g. add `longtable` css class.

Reply:
Sure.